app/page.tsx

```
@@ -7,25 +10,156 @@ import {
   DialogTitle,
   DialogTrigger,
 } from "@/components/ui/dialog";
-
+import { PenIcon } from "lucide-react";
+import PaperForm from "@/components/paperForm";
+import ResearchPaper from "@/components/ResearchPaper";
+interface ResearchPaper {
```

Biome

[error] Shouldn't redeclare 'ResearchPaper'. Consider to delete it or rename it.

'ResearchPaper' is defined here:


(lint/suspicious/noRedeclare)